sklearn/linear_model/tests/test_sgd.py

```diff
@@ -2136,7 +2120,6 @@ def test_SGDClassifier_fit_for_all_backends(backend):
     ],
 )
 def test_loss_deprecated(old_loss, new_loss, Estimator):
-
```

Comment:
In order to preserve git history (e.g. via git-blame), could you revert this change?